### DISCUSSION TITLE
Some GLTF models not visible after upgrade to deck.gl 8.6.0

### DISCUSSION THREAD

**Discussion by euzu** [visgl/deck.gl#6325] [Q&A] [answered]
I have created some 3D models with blender and exported them as glb files. Everything was fine until i upgraded to deck.gl 8.6.0.
Now 9 from 23 models are not visible anymore. I dont get any errors. When i debug, i see that the models are loaded. What could be the issue ?    
I am using the ScreengraphLayer.
I have attached a non visible model.

[default.glb.zip](https://github.com/visgl/deck.gl/files/7417254/default.glb.zip)

**Comment by Pessimistress** [comment, marked as answer]
What version are you upgrading from? Note that loaders.gl core is upgraded to 3.0 in 8.4 and your application should upgrade any loaders dependency accordingly: https://deck.gl/docs/upgrade-guide#loadersgl-v30

@ibgreen

**Reply by euzu** [reply]
I cant upgrade loaders 3.0 because of typescript issues. I cant compile my project with loaders.gl 3.0. The are a lot of typescript compilation problems. I have upgraded from deck.gl 7.  Should i downgrade to deck.gl 8.3 ?

**Reply by euzu** [reply]
Thank you downgrading to 8.3 worked !!!

**Reply by Pessimistress** [reply]
> typescript compilation problems

You can follow the thread here: https://github.com/visgl/loaders.gl/issues/1828

**Reply by euzu** [reply]
thnak you, i saw this thread, it is wip.
